pytests/tests/test_datetime.py

```diff
@@ -227,7 +227,7 @@ def test_datetime_typeerror():
 
 
 @given(dt=st.datetimes(MIN_DATETIME, MAX_DATETIME))
-@example(dt=pdt.datetime(1970, 1, 2, 0, 0))
+@example(dt=pdt.datetime(1971, 1, 2, 0, 0))
```

Comment:
I had to do this to stop it from raising OSErrors on windows...not sure what's with that.

Reply:
Possibly related to #1339?